packages/core/addon/models/fragments/presentation.js

```diff
@@ -1,13 +1,29 @@
 /**
- * Copyright 2017, Yahoo Holdings Inc.
+ * Copyright 2019, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 
 import DS from 'ember-data';
 import Fragment from 'ember-data-model-fragments/fragment';
+import { fragmentArray } from 'ember-data-model-fragments/attributes';
 
 export default Fragment.extend({
   columns: DS.attr('number', { defaultValue: 12 }),
-  layout: DS.attr({ defaultValue: () => [] }),
-  version: DS.attr('number', { defaultValue: '1' })
+  layout: fragmentArray('fragments/layout', { defaultValue: () => [] }),
+  version: DS.attr('number', { defaultValue: '1' }),
+
+  /**
+   * Clones presentation model
+   *
+   * @returns {Fragment} - Ember model data fragment clone of current presentation
+   */
+  clone() {
```

Comment:
Looks we use `copy` in other places:

https://github.com/yahoo/navi/blob/4a152f9b9dd03662cb2de5d2ce2fa089a002d244/packages/core/addon/models/bard-request/fragments/metric.js#L43-L47